cache/lru_cache.cc

```diff
@@ -345,15 +371,16 @@ Status LRUCacheShard::Insert(const Slice& key, uint32_t hash, void* value,
   e->SetInCache(true);
   e->SetPriority(priority);
   memcpy(e->key_data, key.data(), key.size());
+  size_t total_charge = charge + CalcMetadataCharge(e);
```

Comment:
Is looks like that the only time CalcMetadataCharge(e) is not added to e->charge is here, and here charge should be equal to e->charge too. Can we put the `e->charge +` part into CalcMetadataCharge() to make it something like GetTotalCharge() or something like that?